python/ray/worker.py

```diff
@@ -1241,7 +1241,10 @@ def connect(node,
         runtime_env.upload_runtime_env_package_if_needed(job_config)
     elif mode == WORKER_MODE:
         runtime_env.ensure_runtime_env_setup(
-            worker.core_worker.get_job_config().runtime_env)
+            os.environ.get(
+                "RAY_RUNTIME_ENV_FILES",
+                worker.core_worker.get_job_config()
+                .runtime_env.working_dir_uri))
```

Comment:
This is a temporary hack since runtime_env support doesn't support passing per-task files yet.